src/Uvweb/UvBundle/Resources/views/Uv/detail.html.twig

```diff
@@ -20,6 +20,37 @@
 }
 {% endblock %}
 
+{% block js %}
+	{{ parent() }}
+	<script src="//cdnjs.cloudflare.com/ajax/libs/raphael/2.1.0/raphael-min.js"></script>
+	<script src="//cdnjs.cloudflare.com/ajax/libs/morris.js/0.5.1/morris.min.js"></script>
+	<script>
+		(function () {
+
+		{% for criteria, rates in averageCriterias %}
+			Morris.Donut({
```

Comment:
I would prefer to have that in a javascript function which takes`criteria` and `rates` as a parameter.
The goal being to have twig variables only as parameters of a function and not being used inside it.